cli/cdk/client/go/client.go

```diff
@@ -0,0 +1,214 @@
+package cdk
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"time"
+
+	cdk "github.com/lacework/go-sdk/cli/cdk/go/proto/v1"
+	"github.com/lacework/go-sdk/lwlogger"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type ComponentCDKClient struct {
+	logger           ComponentCDKLogger
+	coreClient       cdk.CoreClient
+	conn             *grpc.ClientConn
+	componentVersion string
```

Comment:
Should we know the component name too?

Reply:
The version is a part of the feature data that we inject in the original implementation so that has been carried over here. 

That said, is there are particular way you think we might use the name? Wouldn't be hard to add!

Reply:
Oh, never mind. The component command has already been logged I guess.